pubsub/gcppubsub/gcppubsub.go

```diff
@@ -72,8 +75,8 @@ func SubscriberClient(ctx context.Context, conn *grpc.ClientConn) (*raw.Subscrib
 // TopicOptions will contain configuration for topics.
 type TopicOptions struct{}
 
-// OpenTopic opens the topic on GCP PubSub for the given projectID and
-// topicName.
+// OpenTopic creates a *pubsub.Topic backed by GCP PubSub for the given projectID
```

Comment:
This is a little misleading, because no backend resources are created. Does blob use the same phrase?

Reply:
Good catch; `blob` uses "returns". Updated to use "returns" and also to clarify that the GCP pubsub topic must already exist.